cpp/create_module/algorithm/create.cpp

```diff
@@ -1 +1,50 @@
 #include "create.hpp"
+
+void Create::SetProperties(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
+  const auto arguments = mgp::List(args);
+  const auto graph = mgp::Graph(memgraph_graph);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    std::set<mgp::Id> nodeIds;
+
+    if (arguments[0].IsList()) {
+      for (const auto element : arguments[0].ValueList()) {
+        if (element.IsNode()) {
+          nodeIds.insert(std::move(element.ValueNode().Id()));
+        } else if (element.IsInt()) {
+          nodeIds.insert(std::move(mgp::Id::FromInt(element.ValueInt())));
+        } else {
+          throw mgp::ValueException("First argument must be a node, node's id or a list of those.");
+        }
```

Comment:
I see your comment that you can isolate this part into function but can you explain about type casting? Can you just send mgp::Value and then check what you got there?